components/tooltip/test/tooltip.test.js

```diff
@@ -112,7 +113,7 @@ describe('d2l-tooltip', () => {
 	describe('show-hide', () => {
 
 		it('should show when target is focused', async() => {
-			await triggerFocusFor(tooltipFixture.querySelector('#explicit-target'));
+			await focusWithKeyboard(tooltipFixture.querySelector('#explicit-target'));
```

Comment:
Do you want the `setTimeout` here as well?

Reply:
Tooltip does a lot of internal `await`-ing to update its position and stuff before firing its event, so technically no we don't need anything here.